pkgs/by-name/ba/basedpyright/package.nix

```
@@ -98,14 +98,15 @@ buildNpmPackage rec {
 
   postPatch = ''
     chmod +w ../../
-    mkdir ../../docstubs
-    ln -s ${docstubs}/stubs ../../docstubs
+    ln -s ${pyright-docstubs}/ ../../docstubs
     ln -s ${pyright-root}/node_modules ../../node_modules
     chmod +w ../pyright-internal
     ln -s ${pyright-internal}/node_modules ../pyright-internal/node_modules
   '';
 
   postInstall = ''
+    # unlink $out/lib/node_modules/basedpyright/dist/typeshed-fallback
+    # ln -s ${pyright-docstubs} $out/lib/node_modules/basedpyright/dist/typeshed-fallback
```

```suggestion
    # unlink $out/lib/node_modules/basedpyright/dist/typeshed-fallback
    # ln -s ${pyright-docstubs} $out/lib/node_modules/basedpyright/dist/typeshed-fallback
```

Why not just remove those ?